aspnetcore/security/authentication/sociallogins/sample/Startup.cs

```diff
@@ -87,6 +87,24 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 AppSecret = Configuration["Authentication:Facebook:AppSecret"]
             });
 
+            app.UseTwitterAuthentication(new TwitterOptions()
```

Comment:
Given the choice of:

1. Taking all provider setup code out of Startup.cs
2. Taking most provider setup code out of Startup.cs and leaving just one example (Facebook)
3. Commenting-out provider setup in Startup.cs

I would prefer the first option, and then change all provider setup code snippets in the topics from linked code to snap-shotted inline code. Going to make the change tonight or tomorrow.